### PR TITLE
Add change log for the 3.0.0 release

### PR DESCRIPTION
**Pull request by ZeroPointEnergy**
Added Changelog with the most notable changes since the 2.0.0-beta2. Let me know if I have forgotten something. I will leave this open for comment for a while and if everyone is ok I will finally make the 3.0.0 release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Change Log
+
+Notable changes to new releases of Puppet Dashboard.
+
+## [3.0.0] - unreleased
+
+This release includes changes from over 5 years since the 2.0.0-beta2.
```
Comment (bwitt) [inline CHANGELOG.md]:
```suggestion
This release includes changes from over 5 years since the 2.0.0-beta2 release.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- Handle delayed_job failures better
+- Various enhancements to sorting in various views
+- Improvements to error detection in the log
+- Partial radiator reload
+- Various fixes to various tasks
+
+### Changed
+
+- Updated Rails framework to 5.2.3
+- Support for new Ruby versions (2.4, 2.5, 2.6)
```
Comment (bwitt) [inline CHANGELOG.md]: maybe we should talk about removing support for old versions?

**Comment by ZeroPointEnergy** [comment]
I agree with your suggestions, added them. Also stated the reason why we had to remove support for older ruby version.

**Comment by bwitt** [comment]
thanks lgtm!

**Comment by ZeroPointEnergy** [comment]
Ok, I think this was enough time for objections. Going to do the release now

**Pull request merged**